rocketpy/mathutils/function.py

```diff
@@ -93,12 +95,18 @@ def __init__(
 
         Notes
         -----
-        (I) CSV files can optionally contain a single header line. If present,
-        the header is ignored during processing.
+        (I) CSV files can optionally contain a single header line. If a header
+        naming each data column is present, it will be used for naming the `inputs`
+        and `outputs` left `None`. If the header is not formatted in this way it
+        will be ignored.
+
```

Comment:
This is a little confusing. Suggestion:

```
CSV files may include an optional single header line. If this 
header line is present and contains names for each 
data column, those names will be used to label the inputs
and outputs unless specified otherwise. If there are header 
specified for only a few columns, the remaining unnamed
columns will have the string `None` assigned to it.
```

Reply:
Thanks for the suggestion. I have reworded it a bit from your comment. Actually if the number of "header columns" does not match the number of inputs/outputs, the header is ignored.